sustainml_docs/rst/developer_manual/configuration_request/configuration_request.rst

```diff
@@ -0,0 +1,27 @@
+.. _node_config_service:
+
+Node Configuration Service
+--------------------------
+
+.. image:: ../../figures/SustainML-Framework-Service_design.png
+    :align: center
+    :scale: 75%
+
+This service enables **real-time, dynamic configuration and information request** of any module node in the system.
+For details on the service request and request types specification in the IDL,
+please refer to the :ref:`Service Types <service_types>`.
+
+The orchestrator exposes a **synchronous RPC** over DDS via the `configuration_request()` API.
```

Comment:
Pls add the complete acronym Remote Procedure Call (RPC) the first time naming it